packages/site/src/data/angular/libraries.ts

```diff
@@ -618,4 +618,16 @@ export const libraries: Library[] = [
 			'Storybook is a development environment for UI components. It allows you to browse a component library, view the different states of each component, and interactively develop and test components.',
 		language: 'NodeJS',
 	},
+	{
+		name: 'Ignite UI',
+		author: 'Infragistics',
+		repo: 'https://github.com/IgniteUI/igniteui-angular',
+		package: 'https://www.npmjs.com/package/igniteui-angular',
+		href: 'https://www.infragistics.com/products/ignite-ui-angular',
+		image: 'https://static.infragistics.com/marketing/Website/products/ignite-ui-landing/ignite-ui-logo.svg',
```

Comment:
Let's go with this image instead which just shows the rocket without the text. 
It will look better in the card here

<img width="429" alt="Screenshot 2023-03-17 at 12 13 57 PM" src="https://user-images.githubusercontent.com/67210629/225998487-7716ca8c-24fe-4fc5-868e-df1debc685c4.png">

```suggestion
		image: 'https://avatars.githubusercontent.com/u/5366066?s=200&v=4',
```